portfolio/src/main/webapp/map.js

```diff
@@ -27,6 +27,17 @@ function storeCoords(lat, lng){
     localStorage.setItem("pathList", JSON.stringify(pathList));
 }
 
+function removeCoord(lat, lng){
+    var position = {lat: lat, lng: lng};
+    for (i = 0; i < pathList.length; i++) {
+        if ((position.lat == pathList[i].lat) && (position.lng == pathList[i].lng)){
+            console.log("true");
+            pathList.splice(i, 1);
+        }
+    }
+    localStorage.setItem("pathList", JSON.stringify(pathList));
```

Comment:
side note: I completely forgot you've been using local storage. That'd be a reasonable way to do the "click an event to get to a page for that specific event" thing, right? URL params are probably a better long term solution because it means you can copy/paste the URL to share the event with someone, but since you've added all of these examples of local storage, that may be a good way to go